docs/uuid.md

```diff
@@ -41,7 +35,7 @@ This script will add a UUID to the start of every JSON file that doesn't current
 To use the script, run:
 
 ```sh
-cd <path_to_PrairieLearn>
+cd <path_to_PrairieLearn_source_code>
 cd tools
```

Comment:
```suggestion
cd contrib
```

That said... what's the utility of this script? Do you think people actually use it?

Reply:
Can we put this information on a README in `contrib`, and splice that readme into the docs, and link to the spliced README from the uuid page?

Reply:
I just removed it entirely. I don't think anyone is using this script 😅